<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SHREETAMA" failures="2" tests="12" name="Default test" time="116.754" errors="0" timestamp="2025-07-30T17:50:01 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.124"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.624"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.517"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.811"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.632"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.997"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.783"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.505"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.008"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="11.341">
    <failure type="java.lang.AssertionError" message="Card number field is not reset expected [] but found [LCN_004]">
      <![CDATA[java.lang.AssertionError: Card number field is not reset expected [] but found [LCN_004]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at com.LibraryManagement.cucumber.stepDefinition.MembershipSteps.verifyFieldReset(MembershipSteps.java:151)
at ✽.Card number field should be reset(file:///D:/New_Script/Sprint-2/Sprint_2/src/test/java/com/LibraryManagement/cucumber/features/MemberShip.feature:88)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.764">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at com.LibraryManagement.cucumber.stepDefinition.MembershipSteps.duplicateErrorMessage(MembershipSteps.java:164)
at ✽.Error message should appear for duplicate number(file:///D:/New_Script/Sprint-2/Sprint_2/src/test/java/com/LibraryManagement/cucumber/features/MemberShip.feature:102)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.973"/>
</testsuite> <!-- Default test -->
